app/views/components/goodsLocationExpander.scala.html

```diff
@@ -0,0 +1,135 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.gds._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
+
+@this(govukDetails : GovukDetails, link: link, paragraph: paragraphBody, appConfig: AppConfig)
+
+@()(implicit messages: Messages)
+
+@expanderContent = {
+    @paragraph(messages("declaration.goodsLocation.expander.intro"))
+
+    @paragraph(messages("declaration.goodsLocation.expander.paragraph1.title"), "bold")
+    <div id="code-hint" class="govuk-hint">
+    @Html(messages("declaration.goodsLocation.expander.paragraph1.text",
+        link(
+            text = messages("declaration.goodsLocation.expander.paragraph1.link1.text"),
+            call = Call("GET", appConfig.companyInformationRegister),
+            target = "_blank"
+        ).body
+    ))
+    </div>
```

Comment:
Maybe the entire `<div...>...</div>` can be replaced with something like:
```
@govukHint(Hint(
    content = Html(messages("declaration.goodsLocation.expander.paragraph1.text",
        link(
            text = messages("declaration.goodsLocation.expander.paragraph1.link1.text"),
            call = Call("GET", appConfig.companyInformationRegister),
            target = "_blank"
        )
    ))
))
```

Reply:
Done :+1: